Fortnox.NET/Fortnox.NET.csproj

```diff
@@ -2,8 +2,8 @@
 
     <PropertyGroup>
         <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-        <PackageVersion>2.4.1</PackageVersion>
-        <Version>2.4.1</Version>
+        <PackageVersion>2.5.2</PackageVersion>
+        <Version>2.5.2</Version>
```

Comment:
Do we intend to bump both minor and patch version? (I agree that this is a minor version bump, but that would put it at `2.5.1`, right?)

Reply:
I consider this a bug fix for the date formating. The previously merged PR was a minor version bump with new features.

Reply:
ok, so we forgot to bump the minor version last time?

Reply:
Minor version should have been bumped in #26
So I guess yeah we forgot.